### PR TITLE
Add Optional Support for RDC Profiling

### PR DESCRIPTION
**Pull request by JakopinA**

### CODE REVIEW

File: web/models/stable_diffusion/main.py [hunk 1]
```diff
@@ -352,6 +366,9 @@ def stable_diff_inf(
     pil_images = [Image.fromarray(image) for image in images]
     out_img = pil_images[0]
 
+    if (args.enable_rtc_profiling):
```
Comment (pashu123) [inline web/models/stable_diffusion/main.py:369]: Did you test this? At least test running this file?

Comment (pashu123) [reply web/models/stable_diffusion/main.py:369]: The args are present in stable_args which is a different file.

File: web/models/stable_diffusion/main.py [hunk 2]
```diff
@@ -14,6 +14,8 @@
 import time
 import os
 
+from shark.examples.shark_inference.stable_diffusion.main import start_profiling, end_profiling
```
Comment (pashu123) [inline web/models/stable_diffusion/main.py:17]: Please copy these functions here rather than calling these. Both are separate projects.

**Review by pashu123 — CHANGES_REQUESTED** [review]

File: web/models/stable_diffusion/main.py (hunk 1 above)
Comment (JakopinA) [reply web/models/stable_diffusion/main.py:385]: stable_args is one of the files I changed with this. I made some updates, would you mind checking them?

**Pull request closed without merging**